content/news/2024_03_15_GRASS_DOI_number_at_zenodo.md

```diff
@@ -0,0 +1,65 @@
+author: Peter Loewe and the GRASS Development Team
+---
+
+### Cite GRASS GIS and your geospatial data in scientific works - now with Digital Object Identifiers (DOI)!
+
+
+The GRASS GIS project celebrated its 40th birthday last year. Over the decades, 
+GRASS GIS has spread around the globe and has enabled generations of researchers
+to conduct geospatial research in many fields of science.
+
+<a href="/images/news/grass_in_zenodo.png">
+  <img src="/images/news/grass_in_zenodo.png"
+   alt="GRASS GIS 8.3.2 in Zenodo"
+   title="GRASS GIS 8.3.2 in Zenodo"
+   width="65%" style="float:right;padding-left:15px;padding-top:15px">
+</a>
+
+As a new feature, reflecting the current change towards data-driven science and
+the paradigms of **Open Science** (including Open Source) and the **FAIR** principles 
+(Findable, Accessible, Interoperable, Reusable), GRASS GIS software can now be
+referenced by *Digital Object Identifiers (DOI)*. DOI are digital persistent 
+identifiers to reference internet resources in scientific or official publications. 
+DOI can be resolved in web browsers similar to web addresses (URL). 
+While an URL will fail once the content is moved to another internet
+location, DOI will always point to the intended resource, which is a requirement 
+for long term references in scientific works.
+
+DOI allow to reference the whole GRASS GIS project via a concept DOI 
+(<https://doi.org/10.5281/zenodo.5176030>) or particular software versions 
+like the current [GRASS GIS 8.3.2](https://doi.org/10.5281/zenodo.10817962), 
+or older versions going back to [GRASS GIS 7.8.6](<https://doi.org/10.5281/zenodo.5559460>).
+
+In addition, geospatial datasets for GRASS GIS can also be referenced by a DOI.
+See for example the Natural Earth GRASS dataset at <https://doi.org/10.5281/zenodo.3968936>.
+
+<a href="/images/news/grass_in_zenodo_citation.png">
+  <img src="/images/news/grass_in_zenodo_citation.png"
+   alt="Citation of GRASS GIS 8.3.2 in Zenodo"
+   title="Citation of GRASS GIS 8.3.2 in Zenodo"
+   width="35%" style="float:right;padding-left:15px;padding-top:15px">
+</a>
+
+Since DOI can be interlinked by meta data, connected networks of GRASS GIS related 
+resources can be established, linking code, data and additional information, such as
+video recordings about GRASS GIS. Well known examples are the 
+GRASS GIS code [evolution video](https://doi.org/10.5446/14652), 
+[conference recordings](https://doi.org/10.5446/55251) or the historic
+[USACERL GRASS GIS commercial from 1987](https://doi.org/10.5446/12963),
+narrated by William Shatner of Star Trek fame.
+
+These DOI features are based on an underlying integration of the current 
+GRASS GIS code repository on GitHub with the Open Access repository 
+[Zenodo](https://zenodo.org/), which is also based on Open Source.
+
+A GRASS GIS DOI link will always resolve to a landing page within the Zenodo 
+repository, which is linked to the respective GutHub code repository. The 
```

Comment:
Typo `GutHub`, I just mention this in case someone can make a quick fix.

Reply:
Thanks, fixed in #426

Reply:
Great!